extras/kinesis-asl-assembly/pom.xml

```diff
@@ -142,6 +147,15 @@
             <include>*:*</include>
           </includes>
         </artifactSet>
+        <relocations>
+            <relocation>
+              <pattern>com.google.protobuf</pattern>
+              <shadedPattern>kcl.protobuf</shadedPattern>
```

Comment:
to be consistent, the package name should be `${spark.shade.packageName}.kcl.protobuf`(or `kinesis.protobuf`? kcl is not immediately obvious).